pkgs/ok_http/README.md

```diff
@@ -4,6 +4,25 @@
 An Android Flutter plugin that provides access to the
 [OkHttp][] HTTP client.
 
+## Why use `package:ok_http`?
+
+### 👍 Increased compatibility and reduced disk profile
+
+`package:ok_http` is smaller and works on more devices than other packages.
+
+This size of the [example application]() APK file using different packages:
```

Comment:
Is this supposed to link to nothing? Would it be better to link to the `example/` directory?

Reply:
Fixed!